charts/kube-prometheus-stack/README.md

```diff
@@ -76,6 +76,9 @@ helm upgrade [RELEASE_NAME] prometheus-community/kube-prometheus-stack
 With Helm v3, CRDs created by this chart are not updated by default and should be manually updated.
 Consult also the [Helm Documentation on CRDs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions).
 
+CRDs update lead to a major version bump.
+The Chart's [appVersion](./Chart.yaml#L35) refers to the [`prometheus-operator`](https://github.com/prometheus-operator/prometheus-operator/tree/main)'s version with matching CRDs.
```

Comment:
The line number will be changed after suggestions above.

Please also use an absolute URL, since the README.md will be published to external sides as well.